tokens.xml

```diff
@@ -8364,6 +8417,21 @@ A card with morph can be turned face up any time for its morph cost.)</text>
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Ninja Token </name>
+            <prop>
+                <colors>U</colors>
+                <type>Token Creature — Ninja</type>
+                <maintype>Creature</maintype>
+                <cmc>0</cmc>
+                <pt>2/1</pt>
+            </prop>
+            <set picURL="https://cards.scryfall.io/large/front/a/e/aeec04b1-475c-4e55-b72f-327ea5258146.jpg?1730485615">FDN</set>
+            <reverse-related>Kaito, Cunning Infiltrator</reverse-related>
+            <reverse-related>Kaito, Cunning Infiltrator Emblem</reverse-related>
```

Comment:
we prefer related tags on the emblem for references in the same file

Reply:
gotcha. changed.